filament/backend/src/vulkan/VulkanCommands.cpp

```diff
@@ -270,7 +333,94 @@ void VulkanCommands::updateFences() {
     }
 }
 
-} // namespace filament::backend
+void VulkanCommands::pushGroupMarker(char const* str, VulkanGroupMarkers::Timestamp timestamp) {
+#if FILAMENT_VULKAN_VERBOSE
+    // If the timestamp is not 0, then we are carrying over a marker across buffer submits.
+    // If it is 0, then this is a normal marker push and we should just print debug line as usual.
+    if (timestamp.time_since_epoch().count() == 0.0) {
+        utils::slog.d << "----> " << str << "\n" << utils::io::flush;
```

Comment:
You can use utils::io::endl instead of \n + flush

Reply:
done